bin/cron/sync_dropbox

```diff
@@ -38,16 +38,24 @@ FOLDERS = {
 INTERVAL_SECONDS = 5
 TOTAL_SECONDS = 55 - INTERVAL_SECONDS
 
-logger = HighFrequencyReporter.new(Slack, 'sync-dropbox-staging', '/home/ubuntu/dropbox_sync_error_log.csv')
+LOG_DIR = '/home/ubuntu/staging/log'.freeze
+FileUtils.mkdir_p(LOG_DIR) # This directory should always exist, but just in case
+
+logger = HighFrequencyReporter.new(Slack, 'sync-dropbox-staging', File.join(LOG_DIR, 'dropbox_sync_error_log.csv'))
 logger.load # load errors from most recent sync
 
 while (Time.now - SCRIPT_START) < TOTAL_SECONDS
   attempt_start = Time.now
   logger.reset_new_events # reset new events such that they represent one run through all FOLDERS
   FOLDERS.each do |key, value|
     # 'unison' will sync the two folders. It will return true on success and false on failure.
-    # For full details, see unison's man page. Docs are also here: https://www.cis.upenn.edu/~bcpierce/unison/download/releases/stable/unison-manual.html
-    command = "unison /home/ubuntu/Dropbox/Shared/#{key} /home/ubuntu/staging/pegasus/#{value} -silent -ignore \"Name .dropbox\" -auto -perms 0 -dontchmod -log -logfile /home/ubuntu/unison.log"
+    # For full details, see unison's man page. Docs are also here:
+    # https://www.cis.upenn.edu/~bcpierce/unison/download/releases/stable/unison-manual.html
+    command = [
+      "unison /home/ubuntu/Dropbox/Shared/#{key} /home/ubuntu/staging/pegasus/#{value}",
+      '-silent -ignore "Name .dropbox" -auto -perms 0 -dontchmod',
+      "-log -logfile #{File.join(LOG_DIR, 'unison.log')}"
+    ].join(' ')
```

Comment:
I thought breaking this very long string up into multiple lines might make it more readable, but I'm not entirely sure if it worked or not. Let me know if you'd prefer a more minimal change:

```suggestion
    command = "unison /home/ubuntu/Dropbox/Shared/#{key} /home/ubuntu/staging/pegasus/#{value} -silent -ignore \"Name .dropbox\" -auto -perms 0 -dontchmod -log -logfile #{File.join(LOG_DIR, 'unison.log')}"
```

Reply:
I think I prefer it split up, given the escape characters and the string interpolation which adds it's own complexity.